dask/bag/tests/test_bag.py

```diff
@@ -29,9 +29,10 @@
     total_mem_usage,
 )
 from dask.bag.utils import assert_eq
+from dask.dataframe.utils import assert_eq as assert_eq_df
```

Comment:
This `import` requires `pandas` to be installed, which won't always be the case when running `dask.bag` code. Could you move this import inside the body of `test_to_dataframe_optimize_graph` and add a `pytest.importorskip` call similar to this test

https://github.com/dask/dask/blob/af84db66c420e8349aa4caf49c853e440a99dfb0/dask/bag/tests/test_bag.py#L730-L735

That will result in this test being skipped when `pandas` isn't installed